mautic/mautic
# fix #13570 - incorrect banner when multiple theme deletion

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? 5-x         | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #13570 



## Description
This PR fix the issue when deleting multiple themes in the same times using Delete Selected button.
Before, only one theme name was displaying as deleted (the last one of the selected).
After, deleted banner display for all theme previously deleted







---
### 📋 Steps to test this PR:


1. Navigate to Settings > Themes
2. Upload two custom themes on your Mautic instance.
3. In the list of all themes, locate the checkbox beside each theme's name and select it.
4. Click on Deleted selected button to delete them all in one time
<img width="283" alt="image" src="https://github.com/user-attachments/assets/803f050b-f592-4023-a72d-73a9deda75c6">

5. Now banners appear for each theme deleted
<img width="255" alt="image" src="https://github.com/user-attachments/assets/c95a3ad8-bff7-48bf-a0d7-d4773c581549">
